specs/disciplina/witness/api/witness.yaml

```diff
@@ -347,18 +352,44 @@ paths:
                   nextId:
                     $ref: '#/components/schemas/Hash'
 
+  /checkcv:
+    post:
```

Comment:
@Martoon-00 Fair question. Mainly because serializing a FairCV (which might be quite big) into a GET query string seems like a shitty solution. But I agree that the choice of a method is non-RESTy. We can have a middle-ground here and use PUT method, to emphasize the idempotency of it.

Reply:
Right, makes sense. Please leave a comment on it somewhere in the code.
I'm fine with 'POST', since I've heard a lot of people circumventing `GET` restrictions with `POST`, but not with `PUT`.

Reply:
@Martoon-00 why not with `PUT`? It is _semantically_ incorrect to use anything but `GET` here, strictly speaking, but using `PUT` at least signifies that the method is idempotent (and it is), while `POST` says nothing useful about the method.

Reply:
Well, that makes a fair point. Let's really go with `PUT`